README.md

```diff
@@ -1,40 +1,183 @@
-# cordova-paramedic
+[![Build Status](https://travis-ci.org/apache/cordova-paramedic.svg?branch=master)](https://travis-ci.org/apache/cordova-paramedic)
+[![Build status](https://ci.appveyor.com/api/projects/status/iufmfjo0j0dd4c1w?svg=true)](https://ci.appveyor.com/project/ApacheSoftwareFoundation/cordova-paramedic)
 
 > Paramedic • _noun_ provides advanced levels of care at the point of illness or injury, including out of hospital treatment, and diagnostic services
 
-[![Build Status](https://travis-ci.org/apache/cordova-paramedic.svg?branch=master)](https://travis-ci.org/apache/cordova-paramedic)
-[![Build status](https://ci.appveyor.com/api/projects/status/iufmfjo0j0dd4c1w?svg=true)](https://ci.appveyor.com/project/ApacheSoftwareFoundation/cordova-paramedic)
+# Cordova Paramedic (Test Automation)
+
+`cordova-paramedic` is a tool to automate execution of Cordova plugins tests (via [`cordova-plugin-test-framework`](https://github.com/apache/cordova-plugin-test-framework)).
+
+You can use Paramedic to build and run a Cordova app with plugin tests, run these tests on local and remote emulators on [Sauce Labs](https://saucelabs.com/), and report the results. It can be used on a local or Continuous Integration environment.
 
-Runs cordova medic/buildbot tests locally.
+Cordova Paramedic is currently used to automatically run all plugin tests on CI.
 
-See this [workshop instructions for more explanation](https://kerrishotts.github.io/pgday/workshops/2017/campp/testing.html#cordova-paramedic).
+(See this [workshop instructions for some additional explanation](https://kerrishotts.github.io/pgday/workshops/2017/campp/testing.html#cordova-paramedic).)
+
+## Table of Contents
+
+- [Supported Cordova Platforms](#supported-cordova-platforms)
+- [What does it do?](#what-does-it-do)
+- [Installation](#installation)
+- [Usage](#usage)
+  * [Common usages](#common-usages)
+- [Command Line Interface](#command-line-interface)
+  * [What to build and test](#what-to-build-and-test)
+  * [Emulator/Device to use for tests](#emulatordevice-to-use-for-tests)
+  * [Test Result Server](#test-result-server)
+  * [Test Configuration](#test-configuration)
+  * [Sauce Labs](#sauce-labs)
+- [Configuration file](#configuration-file)
+- [API Interface](#api-interface)
+- [Quirks](#quirks)
+
+<!--<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>-->
 
 ## Supported Cordova Platforms
 
 - Android
 - iOS
-- Windows (Windows 8.1, Windows 10 UWP phone,tablet,desktop)
+- Windows
 - Browser
-- OSX
```

Comment:
Confirm?

Reply:
From the code that documents the parameters:  `"--platform PLATFORM : the platform id. Currently supports 'ios', 'browser', 'windows', 'android'.\n" +`. There is no mention of `osx` at all.

I created https://github.com/apache/cordova-paramedic/issues/89 as this should definitely be supported though for a core platform.